### DISCUSSION TITLE
[PSD5] Exercise 2.1

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#138] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210014957-ef8f9d26-608e-4ea4-a398-ed414d581d29.png)

**Comment by Elia-Belli** [comment]
**Forma Canonica**: non minimizzata (spain without the s)

(a) $\sum{(0,2,3)} = \bar A \bar B + A \bar B + A B$
(b) $\sum{(0,7)} = \bar A \bar B \bar C +  A B C$
(c) $\sum{(0,2,4,5,7)} = \bar A \bar B \bar C +  \bar A B \bar C + A \bar B \bar C +  A \bar B C + A B C$
(d) $\sum{(0,1,2,3,8,10,14)} =  \bar A \bar B \bar C \bar D + \bar A \bar B \bar C D + \bar A \bar B C \bar D + \bar A \bar B C D + A \bar B \bar C \bar D + A \bar B C \bar D + A B C \bar D$
(e) $\sum{(0,3,5,6,9,10,12,15)} = \bar A \bar B \bar C \bar D + \bar A \bar B C D + \bar A B \bar C D + \bar A B C \bar D + A \bar B \bar C D + A \bar B C \bar D + A B \bar C \bar D + A B C D$

**Comment by FeddyLix17** [comment]
@Elia-Belli 
(d) tutto corretto tranne m<sub>14</sub> che è ABCD&#x0304; invece di ABC&#x0304;D&#x0304;
per il resto tutto corretto

**Comment by Elia-Belli** [comment]
Corretto, grazie @FedVlogger17

**Comment by niccolo-fato** [comment]
![photo_2024-11-15_19-52-30](https://github.com/user-attachments/assets/7688b92e-0aa6-453a-816d-d9a40bb1b815)
